contributors/design-proposals/admission_control_event_rate_limit.md

```diff
@@ -0,0 +1,149 @@
+# Admission control plugin: EventRateLimit
+
+## Background
+
+This document proposes a system for using admission control to enforce a limit
+on the number of event requests that the API Server will accept in a given time
+slice. In a large multi-tenant cluster, there may be a small percentage of
+tenants that are always in some type of error state, in which each tenant is
```

Comment:
Use something other than tenant please.  Do you mean namespace, controller, kubelet, client-identity, something else?

Reply:
I believe 'tenant' here is fine, as we already use 'multi-tenancy' phrase.

Reply:
Do not use tenant. It is not specific enough in this context. There is no formal tenant concept in Kubernetes today.